README.md

```diff
@@ -23,45 +20,115 @@ Checkout the branch.
+| `expose.nodePort.ports.https.port` | The service port Harbor listens on when serving with HTTPS |`443`|
+| `expose.nodePort.ports.https.nodePort` | The node port Harbor listens on when serving with HTTPS |`30003`|
+| `expose.nodePort.ports.notary.port` | The service port Notary listens on. Only needed when `notary.enabled` is set to `true` |`4443`|
+| `expose.nodePort.ports.notary.nodePort` | The node port Notary listens on. Only needed when `notary.enabled` is set to `true` |`30004`|
+| **Persistence** |
+| `persistence.enabled`     | Enable the data persistence or not | `true` |
+| `persistence.resourcePolicy`     | Setting it to `keep` to avoid removing PVCs during a helm delete operation. Leaving it empty will delete PVCs after the chart deleted | `keep` |
+| `persistence.persistentVolumeClaim.registry.existingClaim`     | Use the existing PVC which must be created manually before bound | |
+|`persistence.persistentVolumeClaim.registry.storageClass`     | Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used(the default). Set it to `-` to disable dynamic provisioning | |
+|`persistence.persistentVolumeClaim.registry.subPath`     | The sub path used in the volume | `registry` |
+|`persistence.persistentVolumeClaim.registry.accessMode`     | The access mode of the volume | `ReadWriteMany` |
+|`persistence.persistentVolumeClaim.registry.size`     | The size of the volume | `5Gi` |
+|`persistence.persistentVolumeClaim.chartmuseum.existingClaim`     | Use the existing PVC which must be created manually before bound | |
+|`persistence.persistentVolumeClaim.chartmuseum.storageClass`     | Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used(the default). Set it to `-` to disable dynamic provisioning | |
+|`persistence.persistentVolumeClaim.chartmuseum.subPath`     | The sub path used in the volume | `chartmuseum` |
+|`persistence.persistentVolumeClaim.chartmuseum.accessMode`     | The access mode of the volume | `ReadWriteMany` |
+|`persistence.persistentVolumeClaim.chartmuseum.size`     | The size of the volume | `5Gi` |
+|`persistence.persistentVolumeClaim.jobservice.existingClaim`     | Use the existing PVC which must be created manually before bound | |
+|`persistence.persistentVolumeClaim.jobservice.storageClass`     | Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used(the default). Set it to `-` to disable dynamic provisioning | |
+|`persistence.persistentVolumeClaim.jobservice.subPath`     | The sub path used in the volume | `jobservice` |
+|`persistence.persistentVolumeClaim.jobservice.accessMode`     | The access mode of the volume | `ReadWriteMany` |
+|`persistence.persistentVolumeClaim.jobservice.size`     | The size of the volume | `1Gi` |
+|`persistence.persistentVolumeClaim.database.existingClaim`     | Use the existing PVC which must be created manually before bound | |
```

Comment:
Shall we document if user chooses to use external DB/Redis the PVC for DB/Redis will be ignored?